Tests/GRPCCodeGenTests/Internal/Translator/TestFunctions.swift

```diff
@@ -31,6 +31,8 @@
 
 import XCTest
 
+@testable import GRPCCodeGen
```

Comment:
Why do we need `@testable` here?

Reply:
because the I am initialising only the ServerCodeTranslator for the tests, which is internal

Reply:
You aren't doing that in this file though

Reply:
right, my mistake, I thought it was the Snippet Tests file. Here we don't need `@testable`